### PR TITLE
#1 #2 Read me What Why How

### PR DESCRIPTION
**Pull request by Cleop**
#2 Adds copy explaining what Redash is and why it can be useful.
#1 Adds copy explaining how to set Redash up on a project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+## Why?
+Technical team members can easily access a project's data writing SQL queries in the command line however for non-technical team members this is not the case. Having access to a project's data is valuable. It allows you to analyse and evaluate users' behaviours in order to make informed decisions about what the project needs going forward.
+
+Dashboards are an appealing means of displaying a project's data in a user-friendly way. Whilst you can custom make a dashboard for your project this can be complex and time-consuming according to the needs of the client. Redash is a pre-made dashboard tool that is easy to connect to your data and offers a lot of inbuilt functionality to help solve these problems.
+
+## What?
+
+Redash is a paid for service that provides an online, up to date, queryable dashboard of a project's data. Prices start from $29 a month with a free 30 day trial, which is useful for those who may be using it as a stop-gap or want to test the waters of whether it will truly benefit them. To follow the instructions in this readme your project must be hosted on Heroku and using PostgreSQL.
```
Comment (iteles) [inline README.md:11]: Worth qualifying these prices with 'as at the time of writing'

**Review by nelsonic — APPROVED** [review]
@Cleop looking great. only "_improvement_" would be to add "semantic linefeeds" to break up the content and make it easier to review (_and modify later_) http://erambler.co.uk/blog/semantic-linefeeds 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+
```
Comment (iteles) [inline README.md:12]: Might also be useful to add examples of use cases we have seen and implemented in the past, I'm happy to provide content here. https://github.com/dwyl/learn-redash/issues/7

**Review by iteles — APPROVED** [review]
This looks great @Cleop, thanks!

Semantic line breaks as Nelson says would be my only suggestion for a next PR!

**Comment by iteles** [comment]
Forgot to hit 'merge' 🙈

**Pull request merged**